### PR TITLE
Add content-type information when uploading to S3

### LINKED ISSUE #73: Images are uploaded without Content-Type
This can potentially cause issues on browser who don't try and interpret images regardless of Content-Type (maily Safari). We should detect the appropriate type from the file's extension and upload the file appropriately.

### PR DESCRIPTION
**Pull request by RodrigoLeiteF**
----

## What issue does this PR close
Closes #73

## Changes Proposed (a list of new changes introduced by this PR)
1. Adds content type information to S3 uploads

## How to test ( a list of instructions on how to test this PR)
1. Run `mix test`
2. Upload a file to the `/upload` endpoint, send a request to the returned URL and confirm the Content-Type header is correct

### CODE REVIEW

**Review by amiskov — CHANGES_REQUESTED** [review]
Tests are passed on my side, but I couldn't check the actual uploading due to my dev setup. Couldn't make it work with AWS on my localhost 🤷🏻‍♂️

The code itself looks good to me, the only one request is moving the test for authorization to the related branch/PR.

File: test/cambiatus_web/schema/resolvers/accounts_test.exs [hunk 1]
```diff
@@ -43,6 +43,40 @@ defmodule CambiatusWeb.Schema.Resolvers.AccountsTest do
       assert response["data"]["signUp"]["reason"] == ""
     end
 
+    test "fails to create a user when it already exists", %{conn: conn} do
```
[the remark this answers is not in the document]
Comment (RodrigoLeiteF) [reply test/cambiatus_web/schema/resolvers/accounts_test.exs]: Good catch

**Review by lucca65 — CHANGES_REQUESTED** [review]
I've deployed this branch to our dev env and it still don't work with mobile phones. not sure why

**Comment by RodrigoLeiteF** [comment]
This should be merged anyway. It doesn't hurt to send the appropriate metadata for images

**Review by amiskov — APPROVED** [review]

**Comment by lucca65** [comment]
I'm not sure of that, have you tested it with Apple Devices?

**Comment by RodrigoLeiteF** [comment]
> I'm not sure of that, have you tested it with Apple Devices?

It won't fix the issue we're having with XES but either way it's beneficial to send the appropriate Content-Type header when working with images. It's part of the HTTP spec after all.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

**Comment by amiskov** [comment]
> the issue we're having with XES

@lucca65 @RodrigoLeiteF is there any description of the issue? Maybe I can help with the investigation?

**Comment by lucca65** [comment]
We are pushing a lot of code without tests lately. I keep finding issues of code without good enough quality that brings more problems than it solves. I feel like even something small can become a headache because we don't test it enough. Like keep from testing on apple devices is a problem. Because a fraction of our user base uses it and 2 devs also use it. I might not hurt, but the problem with XES persist because of that same way of thinking. Lets merge this since we are in a heavy testing phase. But the overall quality of our deliveries is really bad and should be improved

**Review by lucca65 — APPROVED** [review]

**Pull request merged**